CMakeLists.txt

```diff
@@ -7,63 +7,56 @@
 # ======================================================================
 
 
-# use cmake 3.14 or later
-cmake_minimum_required (VERSION 3.12...3.14 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 
-project(larsimdnn LANGUAGES CXX)
-
-# cetbuildtools contains our cmake modules
-find_package(cetbuildtools REQUIRED)
-
-list(APPEND CMAKE_MODULE_PATH $ENV{CANVAS_ROOT_IO_DIR}/Modules $ENV{ART_DIR}/Modules)
+find_package(cetmodules REQUIRED)
+project(larsimdnn VERSION 09.00.00 LANGUAGES CXX)
 
 include(CetCMakeEnv)
 cet_cmake_env()
 
 cet_set_compiler_flags(DIAGS CAUTIOUS
   WERROR
   NO_UNDEFINED
-  EXTRA_FLAGS -pedantic -Wno-unused-local-typedefs
+  EXTRA_FLAGS -Wno-unused-local-typedefs
```

Comment:
I'd leave `-pedantic` in unless there's a specific reason not to.

Reply:
Wasn't sure about that one.  It isn't everywhere. ;)